src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
+        Colors color = Colors.values()[index];
+        return String.valueOf(color);
```

Comment:
The use of `String.valueOf()` is not recommended for getting the String representation of enum constants. It's better to use the `name()` method to ensure you get the correct name of the enum constant.